package.json

```diff
@@ -157,10 +157,10 @@
     "mkdirp": "^0.5.1",
     "mockdate": "^2.0.1",
     "npm-check-updates": "^2.12.1",
-    "postcss": "^6.0.3",
+    "postcss": "^6.0.20",
```

Comment:
Should we also set this as an exact version like `postcss-load`?

Reply:
The loader is just the issue, we could probably remove this bump if we wanted to